src/TinyMDE.js

```diff
@@ -54,10 +54,11 @@ class Editor {
     this.createEditorElement(element);
     // TODO Placeholder for empty content
     this.setContent(
-      props.content ||
-        (this.textarea
-          ? this.textarea.value
-          : "# Hello TinyMDE!\nEdit **here**")
+      typeof props.content === "string"
+        ? props.content
+        : this.textarea
+        ? this.textarea.value
```

Comment:
not sure how you feel about the nested ternary here (and prettier's formatting isn't making it any better) 😬 happy to change this to something a little more verbose if you prefer.

Reply:
I think it's fine